torch/serialization.py

```diff
@@ -470,7 +480,7 @@ def persistent_load(saved_id):
 
     deserialized_storage_keys = pickle_module.load(f)
 
-    offset = f.tell() if f_is_real_file else None
+    offset = f.tell() if _is_real_file(f) else None
```

Comment:
hmmm I must be missing something, but isn't `f_is_real_file` alread `_is_real_file(f)`?

Reply:
Oh I was testing something out, must have forgotten to change it back. Thanks for catching it!